app/screens/MyLocation/__tests__/MyLocationScreen.tests.tsx

```diff
@@ -10,3 +12,27 @@ it("MyLocationScreen renders correctly", () => {
   const tree = create(<MyLocationScreen />).toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+it("Change current location to ireland", () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const useSelectorMock = jest.spyOn(require("react-redux"), "useSelector");
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const useDispatchMock = jest.spyOn(require("react-redux"), "useDispatch");
+
+  useSelectorMock.mockReturnValue(ElectricityType.france);
+
+  const dummyDispatch = jest.fn();
+  useDispatchMock.mockReturnValue(dummyDispatch);
+
+  render(<MyLocationScreen />);
+  fireEvent.press(screen.root.findByProps({ title: ElectricityType.ireland.toUpperCase() }));
```

Comment:
you don't need the extra library, you can use something like : 
`const item = root.findByProps({ title: ElectricityType.ireland.toUpperCase() });`
and then : 
`item.props.onPress();`
See `SocialMedia.test.tx` as reference